### ISSUE TITLE
Deprecate or remove t.conch.insults.client

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4095]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4095|
|Type|task|
|Created|2009-11-13 19:33:55Z|

The docstring of [twisted/conch/insults/client.py](twisted/conch/insults/client.py) says:

```
You don't really want to use this module. Try insults.py instead.
```

So let's delete or at least deprecate this module.

Attachments:

* [4095.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/786/786f43a1d50ea25ab6aa9d35e5e91ba9847caa19/d685fb237e3645e6c4fda09bfc7bc3a186a4aac7.diff) (4293 bytes) - added by cyli on 2010-06-06 06:39:29Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4095 4095
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__easy easy
time__1258140835000000 1258140835000000
changetime__1276052346000000 1276052346000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to z3p|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_2"></a>|@cyli removed owner|
|-|-|

Similar to [#4096](https://github.com/twisted/twisted/issues/4096):  deprecatedModuleAttribute call in __init__.py, and test of the deprecation.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @cyli|
|-|-|

  1. Some files are missing newlines at the end
  1. Instead of _Test deprecation of .._ as test method docstrings, how about something more like _The L{conch.colors} module is deprecated._
  1. It doesn't seem to be the convention to include the replacement API in .removal files, so I guess drop the second part of the text in this one.

Otherwise looks good, please apply.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli set status to `closed`|
|-|-|

(In [29269]) Applied patch 4095.diff from ticket 4095: deprecate twisted.conch.insults.client

Author: cyli
Reviewier: exarkun
Fixes: [#4095](https://github.com/twisted/twisted/issues/4095)

Deprecation of twisted.conch.insults.client and test for said deprecation.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
